rhel8/profiles/ospp.profile

```diff
@@ -198,3 +198,10 @@ selections:
     - mount_option_dev_shm_nodev
     - mount_option_dev_shm_noexec
     - mount_option_dev_shm_nosuid
+    - var_system_crypto_policy=fips
```

Comment:
What is the reason of having FIPS instead of DEFAULT for ospp?

Reply:
It's because OSPP requires FIPS mode.

Reply:
To be consistent with rule `grub2_enable_fips_mode`, selected in `ospp` profile.